cmd/cluster/powervs/create.go

```diff
@@ -51,6 +51,7 @@ func NewCreateCommand(opts *core.CreateOptions) *cobra.Command {
 	cmd.Flags().BoolVar(&opts.PowerVSPlatform.Debug, "debug", opts.PowerVSPlatform.Debug, "Enabling this will print PowerVS API Request & Response logs")
 	cmd.Flags().BoolVar(&opts.PowerVSPlatform.RecreateSecrets, "recreate-secrets", opts.PowerVSPlatform.RecreateSecrets, "Enabling this flag will recreate creds mentioned https://hypershift-docs.netlify.app/reference/api/#hypershift.openshift.io/v1alpha1.PowerVSPlatformSpec here. This is required when rerunning 'hypershift create cluster powervs' or 'hypershift create infra powervs' commands, since API key once created cannot be retrieved again. Please make sure that cluster name used is unique across different management clusters before using this flag")
 	cmd.Flags().BoolVar(&opts.PowerVSPlatform.PER, "power-edge-router", opts.PowerVSPlatform.PER, "Enabling this flag will utilize Power Edge Router solution via transit gateway instead of cloud connection to create a connection between PowerVS and VPC")
+	cmd.Flags().BoolVar(&opts.PowerVSPlatform.GlobalRoutingEnabled, "global-routing-enabled", opts.PowerVSPlatform.GlobalRoutingEnabled, "Enabling this flag will chose global routing mode when creating transit gateway")
```

Comment:
Let's call this as `TransitGatewayGlobalRouting`, it needs to depict that we are configuring transit gateway's global routing!

Reply:
How about ```TransitGatewayGlobalRoutingEnabled``` to depict that this flag will actually enable global routing on transit gateway ?

Reply:
IMO its not aligning with the rest of the vars, lets not use verb in var name!